test/xhtml_test.html

```diff
@@ -14,7 +14,7 @@
   // IE versions <= 8 do not support XHTML. We cannot unconditionally include
   // this iframe, because on browsers that do not support HTML, the iframe can
```

Comment:
I think "browsers that do not support HTML" should be "browsers that do not support XHTML". Can you fix it too? Thanks!